test/integration/app-proxy-manager-agent.yaml.tmpl

```diff
@@ -40,7 +40,7 @@ spec:
             "image": "{{.hub}}/init:{{.tag}}",
             "args": ["-p", "15001", "-u", "1337"],
             "imagePullPolicy": "Always",
-            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] } }
+            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] }, "privileged": true }
```

Comment:
Is there a more specific set of capabilities that would work instead of full privileges? Maybe CAP_SYS_ADMIN (untested)?